content/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners.md

```diff
@@ -135,7 +135,12 @@ apps/ @octocat
 
 # In this example, @octocat owns any file in the `/apps`
 # directory in the root of your repository except for the `/apps/github`
-# subdirectory, as its owners are left empty.
+# subdirectory, as its owners are left empty. Be careful, ownership of
+# `/apps/github` will not revert back to some other owner previously
+# defined. Leaving `/apps/github` without an owner means that no one
+# owns it. Changes to this directory and its files can be introduced
+# with the approval of any user with at least write access to the
+# repository. 
```

Comment:
I think we can simplify this suggestion, as we already talk about precedence and it should be clear enough that no-one owns the file:

> Without an owner, changes to `apps/github` can only be made with the approval of a user who has write access to the repository.

What do you think?

Reply:
> I think we can simplify this suggestion, as we already talk about precedence and it should be clear enough that no-one owns the file:

All I can say here is that, to my team and I (20 devs in total) it was not clear enough and we made this mistake of leaving a critical configuration file without an owner 🤷🏻‍♂️ 

> > Without an owner, changes to `apps/github` can only be made with the approval of a user who has write access to the repository.
> 
> What do you think? The reason I've not made a "suggestion" is because of the comment formatting and you or I would need to edit the file directly, ideally.

Very similar, I would suggest

> Without an owner, changes to `apps/github` can be made with just the approval of a user who has write access to the repository.

---
I can take care of editing once we agree on the wording.

Reply:
Updated in [e08e3ee](https://github.com/github/docs/pull/34698/commits/e08e3ee66877ffecd7d2a8f9ee656ce0c1de93f6). Closing this conversation.